samza/pom.xml

```diff
@@ -27,7 +27,8 @@
     <description>This project contains all the backend jobs of Sunbird Platform.</description>
 
     <modules>
-        <module>indexer</module>
+	<module>jobs-common</module>
```

Comment:
Add test cases. Ensure 90% code coverage.

Reply:
okay, working on it